modules/server-node/prisma-sqlite/schema.prisma

```diff
@@ -184,6 +183,26 @@ model EventSubscription {
 }
 
 model OnchainTransaction {
+  // Response fields
+  id String @id @default(uuid())
+
+  // Channel fields
+  status         String  @default("submitted") // no enums
+  reason         String // no enums
+  error          String?
```

Comment:
Does it make sense to have just 1 error here? Should we not record errors for each transaction attempt ? Or

Reply:
Maybe 1 overall error here and an error column in attempt?